apps/protocol-frontend/src/hooks/useDaoContributionCount.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@tanstack/react-query';
+import { useUser } from '../contexts/UserContext';
+
+const useDaoContributionCount = (args: {
+  startDate: Date;
+  endDate: Date;
+  guildIds?: number[] | null;
```

Comment:
This should be guildId not guildIds

Reply:
also won't be an array of numbers -- should likely just be a `number`